pandas/tests/window/test_ewm.py

```diff
@@ -108,7 +108,7 @@ def test_ewma_halflife_without_times(halflife_with_times):
 @pytest.mark.parametrize("min_periods", [0, 2])
 def test_ewma_with_times_equal_spacing(halflife_with_times, times, min_periods):
     halflife = halflife_with_times
-    data = np.arange(10)
+    data = np.arange(10.)
```

Comment:
IIUC you can revert this and just keep this PR scoped to the Stata change. see https://github.com/pandas-dev/pandas/issues/35426#issuecomment-665036331